lib/modification.js

```diff
@@ -88,19 +88,20 @@ const addDiffToCtx = async function (req) {
 }
 addDiffToCtx._initial = true
 
-const _getOldAndNew = (action, row, key) => {
+const _getOldAndNew = (action, row, key, entity) => {
   let oldValue = action === 'Create' ? null : row._old && row._old[key]
-  if (oldValue === undefined) oldValue = null
-  else if (Array.isArray(oldValue)) oldValue = JSON.stringify(oldValue)
+  if (oldValue === undefined) {
+    oldValue = action === 'Update' && key in entity.keys ? oldValue = row[key]: null    
```

Comment:
```suggestion
    oldValue = action === 'Update' && key in entity.keys ? row[key] : null
```
otherwise it's a double assignment, no?

Reply:
yes yes, typing error